src/ck-ldb/LBManager.C

```diff
@@ -211,10 +214,32 @@ void _loadbalancerInit()
       CkPrintf(
           "Warning: MetaLB is activated. For Automatic strategy selection in MetaLB, "
           "pass directory of model files using +MetaLBModelDir.\n");
-    while (CmiGetArgStringDesc(argv, "+balancer", &balancer, "Use this load balancer"))
-    {
-      if (CkMyRank() == 0)
-        lbRegistry.addRuntimeBalancer(balancer); /* lbRegistry is a static */
+    if (CkMyRank() == 0) {
+      bool TreeLB_registered = false;
+      while (CmiGetArgStringDesc(argv, "+balancer", &balancer, "Use this load balancer")) {
+        if (strcmp(balancer, "GreedyLB") == 0)
+          _lb_args.legacyCentralizedStrategies().push_back("Greedy");
+        else if (strcmp(balancer, "GreedyRefineLB") == 0)
+          _lb_args.legacyCentralizedStrategies().push_back("GreedyRefine");
+        else if (strcmp(balancer, "RefineLB") == 0)
+          _lb_args.legacyCentralizedStrategies().push_back("RefineA");
```

Comment:
RefineA? Typo?

Reply:
I see now there is refine A and refine B. Do we need two? Shouldn't their names be more useful, or possibly merged into one strategy with different configuration options?